specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2025-04-01/capacityReservation.json

```diff
@@ -1002,15 +1064,15 @@
         },
         "sku": {
           "$ref": "./computeRPCommon.json#/definitions/Sku",
-          "description": "SKU of the resource for which capacity needs be reserved. The SKU name and capacity is required to be set. Currently VM Skus with the capability called 'CapacityReservationSupported' set to true are supported. Refer to List Microsoft.Compute SKUs in a region (https://docs.microsoft.com/rest/api/compute/resourceskus/list) for supported values."
+          "description": "SKU of the resource for which capacity needs be reserved. The SKU name and capacity is required to be set. Currently VM Skus with the capability called 'CapacityReservationSupported' set to true are supported. When 'CapacityReservationSupported' is true, the SKU capability also specifies the 'SupportedCapacityReservationTypes', which lists the types of capacity reservations (such as Targeted or Block) that the SKU supports. Refer to List Microsoft.Compute SKUs in a region (https://docs.microsoft.com/rest/api/compute/resourceskus/list) for supported values. **Note:** The SKU name and capacity cannot be updated for Block capacity reservations."
         }
       },
       "allOf": [
         {
           "$ref": "./computeRPCommon.json#/definitions/UpdateResource"
         }
       ],
-      "description": "Specifies information about the capacity reservation. Only tags and sku.capacity can be updated."
+      "description": "Specifies information about the capacity reservation. Only tags and sku.capacity can be updated. Only tags can be updated for Block Capacity Reservation."
```

Comment:
Specifies information about the capacity reservation. sku.capacity cannot be updated for Block Capacity Reservation. Tags can be update for all Capacity Reservation Types.